pyvista/__init__.py

```diff
@@ -35,7 +35,8 @@
 
 # determine if using at least vtk 5.0.0
 if vtk_version_info.major < 5:
```

Comment:
We don't have ancient VTK running in CI so we should avoid checking coverage here (this is unrelated to your PR, just revealed by your changes).
```suggestion
if vtk_version_info.major < 5:  # pragma: no cover
```